### PR TITLE
Add magic comment to freeze String literals

### PR DESCRIPTION
**Pull request by 007lva**
String literals frozen [will be the default in Ruby 3.0](https://bugs.ruby-lang.org/issues/8976).

### CODE REVIEW

File: spec/integration/session_spec.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 # coding: utf-8
```
Comment (twalpole) [inline spec/integration/session_spec.rb]: The `coding` pragma must be on the first line, otherwise it is ignored

Comment (007lva) [reply spec/integration/session_spec.rb]: Good to know that, thanks!

File: spec/integration/session_spec.rb [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+
```
Comment (twalpole) [inline spec/integration/session_spec.rb]: No blank line between the pragmas

**Review by twalpole — CHANGES_REQUESTED** [review]

**Comment by 007lva** [comment]
@twalpole done

**Review by twalpole — APPROVED** [review]

**Comment by twalpole** [comment]
Thanks

**Pull request merged**